src/cmds/qsub.c

```diff
@@ -162,207 +162,179 @@
-int z_opt = FALSE;
-int A_opt = FALSE;
-int P_opt = FALSE;
-int C_opt = FALSE;
-int J_opt = FALSE;
-int M_opt = FALSE;
-int N_opt = FALSE;
-int S_opt = FALSE;
-int V_opt = FALSE;
-int Depend_opt    = FALSE;
-int Interact_opt  = FALSE;
-int Stagein_opt   = FALSE;
-int Stageout_opt  = FALSE;
-int Sandbox_opt   = FALSE;
-int Grouplist_opt = FALSE;
-int Forwardx11_opt = FALSE;
-#ifdef WIN32
-int gui_opt = FALSE;
-#endif
-int Resvstart_opt = FALSE;
-int Resvend_opt = FALSE;
-int pwd_opt = FALSE;
-int cred_opt = FALSE;
-int block_opt = FALSE;
-int relnodes_on_stageout_opt = FALSE;
-int roptarg_inter = FALSE;
+static char *pbs_hostvar = NULL; /* buffer containing ",PBS_O_HOST=" and host name */
+static int pbs_o_hostsize = sizeof(",PBS_O_HOST=") + 1; /* size of prefix for hostvar */
+static char *display; /* environment variable DISPLAY */
+
+static int no_background = 0; /* flag to disable backgrounding */
+static char  roptarg = 'y'; /* whether the job is rerunnable */
+static char *v_value = NULL; /* expanded variable list from v opt */
+static char *v_value_o = NULL; /* copy of v_value before set_job_env() */
+static char *basic_envlist = NULL; /* basic comma-separated environment variables list string */
+static char *qsub_envlist = NULL; /* comma-separated variables list string */
 #ifndef WIN32
-int x11_disp = FALSE;
+static int x11_disp = FALSE; /* whether DISPLAY environment variable is available */
 #endif
-char *v_value = NULL;
-char *v_value_o = NULL;
-char *basic_envlist = NULL;
-char *qsub_envlist = NULL;
 
-/* for saving option booleans */
-int a_opt_o = FALSE;
-int c_opt_o = FALSE;
-int e_opt_o = FALSE;
-int h_opt_o = FALSE;
-int j_opt_o = FALSE;
-int k_opt_o = FALSE;
-int l_opt_o = FALSE;
-int m_opt_o = FALSE;
-int o_opt_o = FALSE;
-int p_opt_o = FALSE;
-int q_opt_o = FALSE;
-int r_opt_o = FALSE;
-int u_opt_o = FALSE;
-int v_opt_o = FALSE;
-int z_opt_o = FALSE;
-int A_opt_o = FALSE;
-int C_opt_o = FALSE;
-int J_opt_o = FALSE;
-int M_opt_o = FALSE;
-int N_opt_o = FALSE;
-int S_opt_o = FALSE;
-int V_opt_o = FALSE;
-int Depend_opt_o = FALSE;
-int Interact_opt_o = FALSE;
-int Stagein_opt_o = FALSE;
-int Stageout_opt_o = FALSE;
-int Sandbox_opt_o = FALSE;
-int Grouplist_opt_o = FALSE;
-int Forwardx11_opt_o = FALSE;
+/* state booleans for protecting already-set options */
+static int a_opt = FALSE;
+static int c_opt = FALSE;
+static int e_opt = FALSE;
+static int h_opt = FALSE;
+static int j_opt = FALSE;
+static int k_opt = FALSE;
+static int l_opt = FALSE;
+static int m_opt = FALSE;
+static int o_opt = FALSE;
+static int p_opt = FALSE;
+static int q_opt = FALSE;
+static int r_opt = FALSE;
+static int u_opt = FALSE;
+static int v_opt = FALSE;
+static int z_opt = FALSE;
+static int A_opt = FALSE;
+static int C_opt = FALSE;
+static int J_opt = FALSE;
+static int M_opt = FALSE;
+static int N_opt = FALSE;
+static int P_opt = FALSE;
+static int R_opt = FALSE;
+static int S_opt = FALSE;
+static int V_opt = FALSE;
+static int Depend_opt    = FALSE;
```

Comment:
A recent update to the PBS Pro coding standards instructs developers to avoid using whitespace to align variable declarations. In this case, there is too much whitespace left of the equal sign. There are many cases of this still present in the current PR. Searching for [space][space] in the file will identify these. There is no current rule for aligning multiple comments, so those may remain aligned if you choose.